types/algoliasearch/tsconfig.json

```diff
@@ -18,7 +18,6 @@
     },
     "files": [
         "index.d.ts",
-        "lite/index.d.ts",
```

Comment:
What does this mean if a user imports `algoliasearch/lite` now? I think I'm a little lost :)

Reply:
Nothing changes for users &mdash; notice that algoliasearch wasn't one of the packages that published a new version. 

In fact, nothing really changes for algoliasearch since its tests already import and test algoliasearch/lite. `lite/index.d.ts` just isn't in tsconfig.json anymore, which means that it isn't automatically published. You either have to test it &mdash; what algoliasearch does &mdash; or add it to OTHER_FILES.txt to indicate that it should be published untested. Only about 100 packages do the latter.

Reply:
Okay, that makes lots of sense, and makes it simpler to add "common" files which aren't exports. Thanks!